flight/tests/coordinate_conversions/unittest.cpp

```diff
@@ -85,3 +85,306 @@ TEST_F(RneFromLLATest, Equator) {
   ASSERT_NEAR(0, Rne[2][1], eps);
   ASSERT_NEAR(0, Rne[2][2], eps);
 };
+
+
+//// Test fixture for LLA lineraization()
+class LLALineraization : public CoordConversion {
+protected:
+  virtual void SetUp() {
+  }
+
+  virtual void TearDown() {
+  }
+};
+
+TEST_F(LLALineraization, Equator_float) {
+  // Test location is Equator
+  float homeLLA_D[] = {0, 0, 0};
+  float dN = 0.0111319490793274;
```

Comment:
It would be helpful to embed the python script used for generating these values.